valhalla/sif/edgelabel.h

```diff
@@ -478,21 +429,92 @@ class EdgeLabel {
   uint32_t internal_turn_ : 2;
   // Flag indicating edge is an unpaved road.
   uint32_t unpaved_ : 1;
+  uint32_t has_measured_speed_ : 1;
   uint32_t spare : 13;
 
   Cost cost_;      // Cost and elapsed time along the path.
   float sortcost_; // Sort cost - includes A* heuristic.
-  float distance_; // Distance to the destination.
+};
 
-  // Was originally used for reverse search path to remove extra time where paths intersected
-  // but its now used everywhere to measure the difference in time along the edge vs at the node
+/**
+ * Derived label class used for recosting paths within the LabelCallback.
+ * transition_cost is added to the label for use when recoting a path.
```

Comment:
nit: 
```suggestion
 * transition_cost is added to the label for use when recosting a path.
```

Reply:
strange must be disabled somehow for this file?

Reply:
doesn't seem so, but :shrug: